tests/mysql/query.rs

```diff
@@ -997,6 +1003,18 @@ fn select_57() {
     );
 }
 
+#[test]
+fn select_58() {
+    assert_eq!(
+        Query::select()
+            .column(Char::Character)
+            .from(Char::Table)
+            .and_where(Expr::col(Char::Character).like("A").escape('\\'))
+            .to_string(MysqlQueryBuilder),
+        r#"SELECT `character` FROM `character` WHERE `character` LIKE 'A' ESCAPE '\'"#
```

Comment:
Can we double confirm that this is the correct syntax? Because it seemed strange to me at first sight that the `\` in `'\'` does not actually escape the next `'`